src/sdk/config_fs.js

```diff
@@ -9,12 +9,37 @@ const nb_native = require('../util/nb_native');
 const native_fs_utils = require('../util/native_fs_utils');
 const nc_mkm = require('../manage_nsfs/nc_master_key_manager').get_instance();
 
+/* Config directory sub directory comments - 
+   On 5.18 - 
+   1. accounts/ will be deprecated
+   2. A new identities/ directory will be created that represents an account identity, 
+      for example -
+      when an account called alice is created (id = 111) and an IAM user called bob (id = 222) 
+      was created for alice, in the FS it will look as the folloing - 
+      2.1. config_dir/identities/111/identity.json -> represents the account properties (alice)
+      2.2. config_dir/identities/111/users/ -> represents the account's iam users
+      2.3. config_dir/identities/111/users/bob.symlink -> config_dir/identities/222/identity.json (an index to another identity)
+      2.4. config_dir/identities/222/bob.json -> represents the account properties (bob)     
```

Comment:
I think it should be
```suggestion
      2.4. config_dir/identities/222/identity.json -> represents the account properties (bob)     
```